_episodes/44.md

```diff
@@ -120,7 +120,7 @@ We also shipped trino-python-client 0.321.0 with the following improvements:
 
 ## What is Metabase
 
-<img src="../assets/images/logos/metabase-small.png" align="right">
+<img src="{{site.url}}/assets/images/logos/metabase-small.png" align="right">
```

Comment:
This is not necessary ... relative URLs work better in fact.

Reply:
I dunno, what if we ever need to change the structure of where these files are located? I feel like it would bork all these links. Also this `{{site.url}}` breaks when running on the netlify preview. So I just removed it and made it a relative link.